org.kde.kontact.json

```diff
@@ -1230,7 +1229,8 @@
                 }
             ],
             "post-install": [
-                "install run_kontact.sh /app/bin/run_kontact.sh"
+                "mv /app/bin/{kontact,kontact-bin}",
```

Comment:
What's the point of this?

You shot yourself on the feet with it so i have to think that it serves some purpose?